pkg/minikube/cluster/mount.go

```diff
@@ -90,6 +90,9 @@ func Mount(r mountRunner, source string, target string, c *MountConfig) error {
 		}
 		return &MountError{ErrorType: MountErrorUnknown, UnderlyingError: errors.Wrapf(err, "mount with cmd %s ", rr.Command())}
 	}
+	if _, err := r.RunCmd(exec.Command("/bin/bash", "-c", fmt.Sprintf("sudo chmod %o %s", c.Mode, target))); err != nil {
+		return &MountError{ErrorType: MountErrorUnknown, UnderlyingError: errors.Wrap(err, "chmod folder")}
```

Comment:
how about we create a unique error code for not being able to chmod "MountFailedChmod"

Reply:
Added!